src/ape/utils.py

```diff
@@ -116,7 +147,10 @@ def get_package_version(obj: Any) -> str:
 
 
 def deep_merge(dict1, dict2):
-    """Return a new dictionary by merging two dictionaries recursively."""
+    """Return a new dictionary by merging two dictionaries recursively.
+
+    Do you need to be explicit in the what is returned onb line 145?
```

Comment:
Ya you can remove this part, add `-> Dict` and add a `Returns:` doc saying like `dict`